src/main/java/DukeException.java

```diff
@@ -0,0 +1,3 @@
+public class DukeException extends Exception {
```

Comment:
since this is a custom exception, please provide Javadoc comments to explain its purpose in the codebase